### PR TITLE
Dap Curriculum Reorganization

### PR DESCRIPTION
**Pull request by JohanBester**
Hi Folks,
I believe the DAP reorganization has been completed successfully.

Kindly please all review this PR ASAP

Please look at structure rather than grammar, spelling, and styling so much (we can work on those types of things for the following cohort).

Check things like the homework activities, SprintPlanningPoints, and that the curriculum lines up with the cohort schedule (I think I got it all but let's not assume).

Kindly please all review this PR ASAP so that we can get this Merged into development by Sunday / Monday.
We need to be able to make a new release for the curriculum before Wednesday.

Thank you all for your collaboration.

### CODE REVIEW

File: Section09/9.1-HistoryRDBMSandNoSQL.ipynb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+ "cells": [
+  {
```
Comment (Danielle0727) [inline Section09/9.1-HistoryRDBMSandNoSQL.ipynb:3]: Only a blank syntax box appears when viewing file.

**Review by Danielle0727 — COMMENTED** [review]
The following would not allow me to leave comments, unless I overlooked something:

Section 7/Resources/sqlite_cheatsheet_python3... (Only able to view in markdown syntax. I was not sure if it is meant to be that way?)

Section 8/8.3-To SQL or No SQL.md (nothing appears in file)

Great job on moving everything around! You guys definitely did a lot!!!

**Review by jamesleemeredith — APPROVED** [review]

**Comment by jamesleemeredith** [comment]
Looks good from what I can tell - great job Johan!

**Comment by matsinet** [comment]
@Danielle0727 My understanding is that both of those lessons are still to be written, so I am sure those are placeholder files.

**Review by matsinet — APPROVED** [review]

**Pull request merged**